src/platform/unix.rs

```diff
@@ -148,11 +170,13 @@ impl<T> EventLoopExtUnix for EventLoop<T> {
         )
     }
 
+    #[cfg(feature = "x11")]
     #[inline]
     fn new_x11() -> Result<Self, XNotSupported> {
         LinuxEventLoop::new_x11().map(wrap_ev)
     }
 
+    #[cfg(feature = "wayland")]
     #[inline]
```

Comment:
swap `#inline` and `#cfg` to follow the rest of the code in other files, apply to all similar things in that file.